sdk/tracing/tracing-opentelemetry/src/main/java/com/azure/tracing/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -6,51 +6,82 @@
 import com.azure.core.util.logging.ClientLogger;
 import com.azure.core.implementation.util.ImplUtils;
 import com.azure.core.util.Context;
-import io.opencensus.trace.AttributeValue;
+
+import com.azure.tracing.opentelemetry.implementation.AmqpPropagationFormatUtil;
+import com.azure.tracing.opentelemetry.implementation.AmqpTraceUtil;
+import com.azure.tracing.opentelemetry.implementation.HttpTraceUtil;
 import io.opencensus.trace.Span;
 import io.opencensus.trace.Span.Options;
 import io.opencensus.trace.SpanBuilder;
 import io.opencensus.trace.Tracer;
 import io.opencensus.trace.Tracing;
+import io.opencensus.trace.SpanContext;
+import io.opencensus.trace.AttributeValue;
+import io.opencensus.trace.Link;
 
 import java.util.Optional;
 
+import static io.opencensus.trace.Link.Type.PARENT_LINKED_SPAN;
+
 /**
  * OpenTelemetry span
  */
 public class OpenTelemetryTracer implements com.azure.core.implementation.tracing.Tracer {
     // Singleton OpenTelemetry tracer capable of starting and exporting spans.
     private static final Tracer TRACER = Tracing.getTracer();
-    private static final String OPENTELEMETRY_SPAN_KEY = com.azure.core.implementation.tracing.Tracer.OPENTELEMETRY_SPAN_KEY;
-    private static final String OPENTELEMETRY_SPAN_NAME_KEY = com.azure.core.implementation.tracing.Tracer.OPENTELEMETRY_SPAN_NAME_KEY;
+
+    // standard attributes with AMQP call information
+    private static final String COMPONENT = "component";
+    private static final String MESSAGE_BUS_DESTINATION = "message_bus.destination";
+    private static final String PEER_ENDPOINT = "peer.address";
 
     private final ClientLogger logger = new ClientLogger(OpenTelemetryTracer.class);
 
     @Override
-    public Context start(String methodName, Context context) {
-        Span parentSpan = (Span) context.getData(OPENTELEMETRY_SPAN_KEY).orElse(TRACER.getCurrentSpan());
-        String spanName = (String) context.getData(OPENTELEMETRY_SPAN_NAME_KEY).orElse(methodName);
-
-        SpanBuilder spanBuilder = TRACER.spanBuilderWithExplicitParent(spanName, parentSpan);
-        Span span = spanBuilder.startSpan();
+    public Context start(String spanName, Context context) {
+        SpanBuilder spanBuilder = startSpanWithExplicitParent(spanName, context);
+        Span span;
+        if (spanName.contains("send")) {
```

Comment:
what if we add receive? will we need to special-case it too? Is there a more explicit way to assign kind to span?